assets/js/base/components/price-slider/index.js

```diff
@@ -0,0 +1,405 @@
+/**
+ * External dependencies
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+import { Fragment, useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { useDebounce } from '@woocommerce/base-hooks';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { constrainRangeSliderValues, formatCurrencyForInput } from './utils';
+
+const PriceSlider = ( {
+	initialMin,
+	initialMax,
+	min,
+	max,
+	onChange,
+	step,
+	currencySymbol,
+	priceFormat,
+	showInputFields,
+	showFilterButton,
+	isLoading,
```

Comment:
What about `disabled` instead of `isLoading`.  Essentially that's the purpose of this variable and `isLoading` is an implementation detail whatever is feeding props to this one.

Reply:
Loading is more appropriate given the placeholder styling?

Reply:
This _only_ place I see it affecting styling is: `isLoading && 'is-loading'`.  Is that what you're referencing?  Everywhere else it appears to be more for controlling disabled behaviour.  I'll let you make the call :)

Reply:
Yeah, we apply CSS loading styles here (placeholder mixin). I think loading state makes more sense than disabled in this case since that causes the range slider etc to not be visible :)